src/themes/custom/styles/theme.scss

```diff
@@ -11,3 +11,4 @@
 @import '../../../styles/bootstrap_variables_mapping.scss';
 @import '../../../styles/_truncatable-part.component.scss';
 @import './_global-styles.scss';
+@import '../../../../node_modules/ngx-ui-switch/ui-switch.component.scss';
```

Comment:
This is not a comment specific to this PR, but more about our process for these SCSS imports. 

It seems **really odd** to me that individual themes don't import the `src/styles/base-theme.scss` by default... as that means anytime we update the `base-theme.scss` we have to **require** that all custom themes also update their `theme.scss`.

Shouldn't we consider changing this @atarix83 and @artlowel ?  Even adding this single import now requires anyone upgrading to 7.3 to ensure they update their `theme.scss`.  It seems like we could avoid that by just having:
```
@import '../../../styles/base-them.scss'
```
(It'd also let us remove a lot of these other imports by default)

Reply:
You can't just import `base-theme.scss`. The reason these files are separate is because the imports have to happen in a  particular order for the themes, which differs from `base-theme.scss`. The order makes it possible for us to override variables. What we _can_ do in order to make this less of a problem is create an scss file in `src/styles` for 3rd party imports, and import that file in `base-theme.scss`, and both theme.scss files. We can add [that noui slider import](https://github.com/DSpace/dspace-angular/blob/3a9783d031bd6940cdb9bcc172fb57cc292e3ffb/src/styles/base-theme.scss#L3) to it as well. That way theme css files will no longer have to change if a new 3rd party css import gets added

Reply:
@artlowel and @atarix83 : I've moved this over into a separate ticket. https://github.com/DSpace/dspace-angular/issues/1679

I still think it'd be very nice to do at some point soon. If either of you want to do it in 7.3 (as I think it's a very small but beneficial change) let me know. Otherwise, we'll move this to the list of tasks for 7.4